catalyst_voices/apps/voices/lib/pages/workspace/workspace_header.dart

```diff
@@ -0,0 +1,230 @@
+import 'package:catalyst_voices/common/ext/space_ext.dart';
+import 'package:catalyst_voices/routes/routes.dart';
+import 'package:catalyst_voices/widgets/buttons/voices_filled_button.dart';
+import 'package:catalyst_voices/widgets/text_field/voices_text_field.dart';
+import 'package:catalyst_voices_assets/catalyst_voices_assets.dart';
+import 'package:catalyst_voices_blocs/catalyst_voices_blocs.dart';
+import 'package:catalyst_voices_brands/catalyst_voices_brands.dart';
+import 'package:catalyst_voices_localization/catalyst_voices_localization.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
+import 'package:catalyst_voices_view_models/catalyst_voices_view_models.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class WorkspaceHeader extends StatelessWidget {
+  const WorkspaceHeader({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: 2,
+      child: Container(
+        padding: const EdgeInsets.symmetric(horizontal: 32),
+        child: const Column(
+          mainAxisSize: MainAxisSize.min,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            SizedBox(height: 32),
+            _TitleText(),
+            SizedBox(height: 48),
+            Row(
+              mainAxisSize: MainAxisSize.max,
+              crossAxisAlignment: CrossAxisAlignment.end,
+              children: [
+                Expanded(
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.start,
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      _SubtitleText(),
+                      SizedBox(height: 16),
+                      _DescriptionText(),
+                      SizedBox(height: 32),
+                      _DraftProposalButton(),
+                    ],
+                  ),
+                ),
+                SizedBox(width: 20),
+                Expanded(
+                  child: Column(
+                    mainAxisAlignment: MainAxisAlignment.end,
+                    crossAxisAlignment: CrossAxisAlignment.end,
+                    children: [
+                      _SearchTextField(),
```

Comment:
Looks like you could just use `Align` instead of a `Column` but no idea how final the UI here is, probably gonna be changed

Reply:
yeah. Its not final. Not sure what will be here